topics/proteomics/tutorials/ml-modeling-of-anti-cancer-peptides/tutorial.md

```diff
@@ -70,24 +68,17 @@ In this tutorial, we used 6 different machine learning algorithms for the predic
 
 A descriptor or feature is the quantitative or a qualitative measure of a property that is associated with a sequence. For example, a chemical compound can be described via its charge chemical formula, molecular weight, number of rotatable bonds, etc. Similarly, several properties can be deduced from the biological sequence that can be utilized to summarise a biological property such as anti-cancer activity. In this example, we utilized CTD descriptors, describes as composition, transition, and distribution descriptors. 
 
-![Alternative text](../../images/PDAUG_ML_1.png "ML algorithms use numerical representation of a sequence-based properties for model building")
+![Peptide descriptors](../../images/PDAUG_ML_1.png "ML algorithms use numerical representation of a sequence-based properties for model building")
+
 
-### Calculating descriptor for ACPs
 
-> ### {% icon hands_on %} Hands-on: Task description
+> ### {% icon hands_on %} Hands-on: Calculating CTD descriptors for ACPs and non-ACPs
 >
-> 1. **PDAUG Sequence Property Based Descriptors** {% icon tool %} with the following parameters:
+> 1. {% tool [PDAUG Sequence Property Based Descriptors](https://toolshed.g2.bx.psu.edu/view/jay/pdaug_sequence_property_based_descriptors/6b6aec52638d) %} with the following parameters:
```

Comment:
this is the link to the tool repository, we need the tool ID. Please see https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links if you have questions about how to obtain that tool ID.

Reply:
I fixed the IDs and pushed the changes. I would like to thank Both the reviewers for your suggestions to improve these tutorials. Thank you.